com.civicrm.hrjobroles/views/include/add_new_role.html

```diff
@@ -85,63 +85,104 @@
               <div class="form-group">
                 <label for="newLocation" class="col-xs-4 control-label">Location:</label>
                 <div class="col-xs-6">
-                  <ui-select
-                    prevent-animations
-                    ng-model="jobroles.editData['new_role_id']['location']"
-                    theme="civihr-ui-select"
-                    title="Choose a location"
-                    name="newLocation">
-                    <ui-select-match prevent-animations allow-clear placeholder="Select a location">{{$select.selected.value.title}}</ui-select-match>
-                    <ui-select-choices prevent-animations repeat="location.value.value as (key,location) in jobroles.LocationsData | filter: { value: { title: $select.search }}">
-                      <div ng-bind-html="location.value.title | highlight: $select.search"></div>
-                    </ui-select-choices>
-                  </ui-select>
+                  <div class="form-inline">
```

Comment:
There are 8 very similar blocks added in this PR. Is there a potential way to reduce the code duplication? Maybe we can create a partial template and simply pass parameters in there?

Reply:
Good idea. I will explore that.